lab4.java

```diff
@@ -0,0 +1,27 @@
+import java.util.Scanner;  
+
+
+public class lab4 {
+
+
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
```

Comment:
Unneccessary line break.
```suggestion
        Scanner scanner = new Scanner(System.in);
```